test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua

```diff
@@ -0,0 +1,49 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when WEB_VIEW AppHmiType is absent in application policies
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table not contain WEB_VIEW.
+--     SDL->appID: REJECT, success:"false":RegisterAppInterface()
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMITypeWebView = "WEB_VIEW"
+local appHMITypeMedia = "MEDIA"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMITypeWebView }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, appHMITypeMedia })
+common.Step("Start SDL, HMI", common.startWOdeviceConnect)
+common.Step("Connect WebEngine device", common.connectWebEngine,
+  { appSessionId, config.defaultMobileAdapterType })
+
+common.Title("Test")
+common.Step("Register App, policy table not contain WEB_VIEW", common.rejectedRegisterApp)
```

Comment:
@ydementieiev `PT does not contain "WEB_VIEW" AppHMIType`

Reply:
@AByzhynar thank you, [changed](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/abede64e2119bad9c40c14ecd9e5ef9e03ce0228)